lms/static/scripts/frontend_apps/components/SubmitGradeForm.js

```diff
@@ -93,6 +99,11 @@ export default function SubmitGradeForm({ disabled = false, student }) {
     setGradeSaved(false);
   }, [student]);
 
+  useLayoutEffect(() => {
+    inputRef.current.focus();
+    inputRef.current.select();
+  }, [grade]);
```

Comment:
Can you help me understand why this triggers on change of `grade` instead of change of `student?`

Reply:
A student change triggers the grade fetch and at that point, we wouldn't have any grade to select. When the fetch completes, the grade gets set and only at that time we have the grade text to select in the input filed. This will get triggered even if 2 different students have the same grade because the grade gets set to '' when changing students. The test actually returns the same grade for both students since fakeFetchGrade does not change.